std_msgs/Header header
geometry_msgs/Vector3[] greens
geometry_msgs/Vector3[] blues
